tests/test_terra_notebook_utils.py

```diff
@@ -54,11 +58,6 @@ def test_oneshot_copy(self):
     def test_multipart_copy(self):
         drs.copy(self.drs_url, "test_oneshot_object", multipart_threshold=1024 * 1024)
 
-    # Probably don't want to run this test very often. Once a week?
```

Comment:
Should this be kept separately and manually run?

Reply:
Perhaps we can use our swanky new CI/CD system to execute this once a week, or once a month.